### ISSUE TITLE
cockpit.service fails to start on non-selinux systems

### ISSUE THREAD

**Issue by mbiebl** [cockpit-project/cockpit#3206]
Version: 0.84

After the update from 0.83 to 0.84, cockpit.service fails to start
```
# systemctl status cockpit.service 
● cockpit.service - Cockpit Web Service
   Loaded: loaded (/lib/systemd/system/cockpit.service; static; vendor preset: enabled)
   Active: failed (Result: exit-code) since Sa 2015-11-21 15:46:20 CET; 1min 38s ago
     Docs: man:cockpit-ws(8)
  Process: 2769 ExecStartPre=/usr/sbin/remotectl certificate --ensure --user=root --group= --selinux-type= (code=exited, status=1/FAILURE)

Nov 21 15:46:20 pluto systemd[1]: Starting Cockpit Web Service...
Nov 21 15:46:20 pluto remotectl[2769]: /usr/bin/chcon: can't apply partial context to unlabeled file ‘/etc/cockpit/ws-certs.d/0-self-signed.cert’
Nov 21 15:46:20 pluto remotectl[2769]: remotectl: couldn't change SELinux type context '' for certificate: /etc/cockpit/ws-certs.d/0-self-signed.cert: Child process exited with code 1
Nov 21 15:46:20 pluto systemd[1]: cockpit.service: Control process exited, code=exited status=1
Nov 21 15:46:20 pluto systemd[1]: Failed to start Cockpit Web Service.
Nov 21 15:46:20 pluto systemd[1]: cockpit.service: Unit entered failed state.
Nov 21 15:46:20 pluto systemd[1]: cockpit.service: Failed with result 'exit-code'.
```
SELinux is not active on this Debian system.

One could use ExecStartPre=-/usr/sbin/remotectl ... to ignore the failure from remotectl, but maybe it's better to make the use of chcon dependend on whether selinux is enabled during runtime

**Comment by mbiebl** [comment]
Already fixed by 6246c16d0bed3ce63f3b598684849823d8661282

**Comment by stefwalter** [comment]
This is a good reason why we need to have Debian testing (in this case non-SELinux system testing) upstream.
